client/src/components/Overview/ProductInformation.jsx

```diff
@@ -22,10 +29,10 @@ function ProductInformation({ product, styles, selectedStyleId, handleStyleSelec
       Style &gt; {selectedStyle?.name}
       <br />
       {/*TODO: add strikethrough for sale*/}
-      {selectedStyle?.original_price}
-      {selectedStyle?.sale_price !== 0 ? selectedStyle?.sale_price : null}
       {/*TODO: show Product Overview*/}
       {/*TODO: add Share buttons*/}
+      {selectedStyle?.original_price}
+      {selectedStyle?.sale_price !== 0 ? selectedStyle?.sale_price : null}
```

Comment:
Seems this is specific to one product, so maybe `<StyleSelector>` shouldn't be inside of it. Would help with not having to pass down so many props to so many descendants